### ISSUE TITLE
Support for hosting static assets

### ISSUE THREAD

**Issue by judepereira** [storyscript/community#10]
Perhaps we should consider adding the ability for hosting static assets? @sawthinkar from the public Slack channel had a request for this.

**Comment by judepereira** [comment]
// cc @rohit121 @stevepeak @williammartin

**Comment by stevepeak** [comment]
Yes, I like this feature and I have a feeling it will be quite demanded. Let's plan and POC it out.

**Comment by judepereira** [comment]
I'll plan this out next week. @rohit121 can you add this to the next sprint please?
